snippets/about-crio-snippet.adoc

```diff
@@ -0,0 +1,9 @@
+// Text snippet included in the following modules:
+//
+// * modules/installation-custom-aws-vpc.adoc
+// * modules/installation-about-custom-azure-vnet.adoc
+// * modules/installation-custom-gcp-vpc.adoc
+
+:_content-type: SNIPPET
+
+CRI-O is a Kubernetes-native container engine implementation that integrates closely with the operating system to deliver an efficient and optimized Kubernetes experience. The CRI-O container engine is managed using a systemd service on each {product-title} cluster node.
```

Comment:
This is [existing text](https://docs.openshift.com/container-platform/4.11/support/troubleshooting/troubleshooting-crio-issues.html#about-crio_troubleshooting-crio-issues).

Reply:
There's a lack of agency.

> Each {product-title} cluster node manages an instance of the CRI-O container engine as a systemd service.

Or similar. Just a thought.